package.json

```diff
@@ -164,13 +156,12 @@
         "less": "^3.11.3",
         "mocha": "^7.2.0",
         "node-loader": "^2.0.0",
-        "npm-force-resolutions": "0.0.10",
         "prettier": "^2.0.5",
         "style-loader": "^1.2.1",
         "svg-inline-loader": "^0.8.2",
         "thread-loader": "^3.0.4",
-        "ts-loader": "^7.0.5",
-        "typescript": "^3.9.3",
+        "ts-loader": "^9.3.1",
+        "typescript": "^4.7.4",
```

Comment:
Had to update ts as well, due to new vscode dependencies (the *.d.ts files require later versions of TS)